src/app/(root)/deployments/[...id]/page.tsx

```diff
@@ -14,6 +14,6 @@ export default function InteractionsPage({
         id: string[];
     };
 }) {
-    const deploymentId = params.id[0];
-    return <InteractionClient initialId={deploymentId} />;
+    const baseTokenAddress = params.id[0];
```

Comment:
Shouldn't this be the gluon reactor's contract address instead of the base token address?  There could be many deployed reactors using the same base token.

Reply:
Yes, you are correct. I have changed it back to `deploymentAddress`.